libraries/protocol/steem_operations.cpp

```diff
@@ -631,6 +631,39 @@ namespace steem { namespace protocol {
       FC_ASSERT( reward_steem.amount > 0 || reward_sbd.amount > 0 || reward_vests.amount > 0, "Must claim something." );
    }
 
+#ifdef STEEM_ENABLE_SMT
+   void claim_reward_balance2_operation::add_reward_token( const asset& reward_token )
```

Comment:
I suppose this is useful for testing, but it doesn't belong on the operation. As far as I can tell, it isn't being used anywhere outside of the tests and will not be called in the normal validation/evaluation of the operation.

In fact, this is very bad. The function is only used in testing and is tested in testing. All operation testing should be done using `validate` and by pushing the transaction to the blockchain.

Reply:
Removed.